src/azure-cli/azure/cli/command_modules/synapse/manual/operations/artifacts.py

```diff
@@ -465,3 +467,88 @@ def create_or_update_spark_job_definition(cmd, workspace_name, spark_job_definit
     properties = SparkJobDefinition.from_dict(definition_file)
     return sdk_no_wait(no_wait, client.begin_create_or_update_spark_job_definition,
                        spark_job_definition_name, properties, polling=True)
+
+
+def list_sql_scripts(cmd, workspace_name):
+    client = cf_synapse_sql_script(cmd.cli_ctx, workspace_name)
+    return client.get_sql_scripts_by_workspace()
+
+
+def get_sql_script(cmd, workspace_name, sql_script_name):
+    client = cf_synapse_sql_script(cmd.cli_ctx, workspace_name)
```

Comment:
```suggestion
def get_sql_script(cmd, workspace_name, sql_script_name):
    client = cf_synapse_sql_script(cmd.cli_ctx, workspace_name)
```
Same as above, `g.show_command('show', 'get_sql_script')` should work.